airbyte-integrations/connector-templates/source-python-http-api/setup.py.hbs

```diff
@@ -6,7 +6,7 @@
 from setuptools import find_packages, setup
 
 MAIN_REQUIREMENTS = [
-    "airbyte-cdk~=0.1",
+    "airbyte-cdk~=0.1.56",
```

Comment:
Should this be a variable to always point to the latest version?  This assumes that all new connectors should use the latest version, which seems right to me (this is the template)

Reply:
I'm not sure what you're suggesting here - do you mean moving the version specification to the `plopfile.js` and passing it in to the generated templates?

I think we would also need to do some automation or something to get it to always point to the latest version though 🤔

Reply:
Yep - I'm suggestion that perhaps we want that automation

Reply:
Got it! Makes sense. I wasn't able to get to this today but want to get this in to avoid people getting started from running into this, so will merge this as is. Created #12859 for the automation that we can chat about / prioritize accordingly.